motioneye/templates/main.html

```diff
@@ -161,6 +161,11 @@
                         <span class="minimize open"></span>
                     </div>
                     <table class="settings">
+                        <tr class="settings-item" strip="true">
+                            <td class="settings-item-label"><span class="settings-item-label">{{ _("Lingvo") }}</span></td>
+                            <td class="settings-item-value"><input type="text" autocapitalize="none" class="styled general main-config" id="langEntry"></td>
```

Comment:
I suggest to make this a selection/list with all supported languages instead of an input box. So users do not need to guess which language is supported and how to enter it correctly (like `en` for English).